checker/src/main/java/org/checkerframework/checker/optional/guava-assertions.astub

```diff
@@ -0,0 +1,353 @@
+// TODO: This file does not belong in the Optional Checker.  It should be in checker.jar in a
```

Comment:
This need to be in `checker/src/main/resources`.  You (or I can do it later) should probably add the annotations in `guava/src/com/google/common/base/Verify.java`, too. (https://github.com/typetools/guava/commit/68e7ba4b9b40083954d04678233e19fb51649818#diff-3b41ae59f30981b976cc5644298f5b9950e3192599e06ef1ce5512cbbad57161). And the annotations in this file should be kept in sync with the typetools/guava annotations.

Reply:
I've done this here: https://github.com/typetools/checker-framework/pull/6365